package.json

```diff
@@ -42,8 +44,7 @@
     "button"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.6",
-    "ember-cli-htmlbars": "1.0.3"
```

Comment:
Fresh ember addons are generated that way. https://github.com/ember-cli/ember-addon-output/blob/master/package.json#L29

I couldn't remember if we differed for a reason